easybuild/easyconfigs/w/WRF/WRF-3.6.1-CrayGNU-2015.11-XC-dmpar.eb

```diff
@@ -0,0 +1,41 @@
+name = 'WRF'
+version = '3.6.1'
+
+homepage = 'http://www.wrf-model.org'
+description = """The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale
+ numerical weather prediction system designed to serve both operational forecasting and atmospheric
+ research needs."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11-XC'}
+toolchainopts = {'opt': False}  # don't use agressive optimization, stick to -O2
+
+sources = ['%(name)sV%(version)s.TAR.gz']
+source_urls = [
+    'http://www2.mmm.ucar.edu/wrf/src/',
+    'http://www.mmm.ucar.edu/wrf/src/',
+]
+
+# csh is used by WRF install scripts
+builddependencies = [('tcsh', '6.18.01')]
```

Comment:
on Sisu, this `tcsh` is crashing (segfaulting) when we use it to build WRF

previously, we've simply relied on the OS-provided `tcsh` to fix this issue, but maybe we should try and figure out what's wrong with the tcsh build there...